docs/conf.py

```diff
@@ -16,36 +14,15 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
     "sphinxcontrib.autoprogram",
+    "towncrier_draft_ext",  # in-tree
 ]
-ROOT_SRC_TREE_DIR = Path(__file__).parents[1]
-
-
-def generate_draft_news():
-    home = "https://github.com"
-    issue = "{}/issue".format(home)
-    fragments_path = ROOT_SRC_TREE_DIR / "docs" / "changelog"
-    for pattern, replacement in (
-        (r"[^`]@([^,\s]+)", r"`@\1 <{}/\1>`_".format(home)),
-        (r"[^`]#([\d]+)", r"`#pr\1 <{}/\1>`_".format(issue)),
-    ):
-        for path in fragments_path.glob("*.rst"):
-            path.write_text(re.sub(pattern, replacement, path.read_text()))
-    env = os.environ.copy()
-    env["PATH"] += os.pathsep.join(
-        [os.path.dirname(sys.executable)] + env["PATH"].split(os.pathsep),
-    )
-    changelog = subprocess.check_output(
-        ["towncrier", "--draft", "--version", "DRAFT"], cwd=str(ROOT_SRC_TREE_DIR), env=env,
-    ).decode("utf-8")
-    if "No significant changes" in changelog:
-        content = ""
-    else:
-        note = "*Changes in master, but not released yet are under the draft section*."
```

Comment:
@gaborbernat the only difference is that the extension does not implement this (I simply missed it). Is it vital if this line is missing?